bmq-sdk/src/test/java/com/bloomberg/bmq/it/util/BmqBroker.java

```diff
@@ -23,10 +23,11 @@
 import java.util.UUID;
 
 public interface BmqBroker extends TestTcpServer {
+    int BROKER_DEFAULT_PORT = 30114;
 
     enum Domains {
-        Priority("bmq.test.mmap.priority"),
-        Fanout("bmq.test.mmap.fanout");
+        Priority("bmq.test.mem.priority"),
+        Fanout("bmq.test.mem.fanout");
```

Comment:
We don't have `mmap` domains in the base BlazingMQ repo, so I changed it to similar `mem` domains. It will make docker and native launch similar.